app/src/main/java/com/github/h3lp3rs/h3lp/HelpParametersActivity.kt

```diff
@@ -127,11 +126,11 @@ class HelpParametersActivity : AppCompatActivity() {
      */
     private fun launchEmergencyCall() {
         calledEmergencies = true
-        updateCoordinates()
+        locationHelper.updateCoordinates(this)
```

Comment:
Maybe add a comment on why in the case of launching an emergency, we get the location again (since we already got it in onCreate) which I guess is to have the most accurate location at the time of sending the emergency